Build.PL

```diff
@@ -14,6 +14,12 @@ use builder::custom;
 use File::Basename;
 use File::Spec;
 
+my $tap_harness_args = {
+    "verbosity" => 1,
+    "merge" => 1,
+    "formatter" => "TAP::Formatter::GitHubActions",
```

Comment:
```suggestion
    "formatter_class" => "TAP::Formatter::GitHubActions",
```

`formatter` key expects a `TAP::Formatter::*` object as value.